### PR TITLE
add -vendor flag to include/exclue vendor directory to/from scanning

### PR DESCRIPTION
**Pull request by krhubert**
There is too much output with vendoring tools, that is not related to ones repo

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -164,6 +167,9 @@ func CrawlPaths(paths []string) chan string {
 				continue
 			}
 			filepath.Walk(path, func(path string, info os.FileInfo, err error) error {
+				if !*vendor && (strings.HasPrefix(path, "vendor/") || strings.Contains(path, "/vendor/")) {
```
Comment (shurcooL) [inline main.go]: `filepath.Walk` deals with system filepaths, and the separator is not guaranteed to be `/` on every OS. As a result, checking if `strings.HasPrefix(path, "vendor/")` or `strings.Contains(path, "/vendor/") is not correct.

Comment (krhubert) [inline main.go]: hum right, I'll change this to filepath.Separator

**Comment by mibk** [comment]
You are right about that. It would be helpful to ignore vendor directory by default. I'll look at it.

**Comment by mibk** [comment]
@krhubert Thank you very much!

**Comment by krhubert** [comment]
@mibk Thanks too for review and merge!

**Pull request merged**